apps/src/storage/dataBrowser/LibraryTable.jsx

```diff
@@ -35,19 +74,28 @@ class LibraryTable extends React.Component {
     return (
       <div>
         <a style={styles.tableName} onClick={this.toggleCollapsed}>
-          <FontAwesome icon={icon} />
+          <FontAwesome className="fa fa-fw" icon={icon} />
           <span>{this.props.name}</span>
         </a>
         {!this.state.collapsed && (
-          <div>
+          <div style={styles.collapsibleContainer}>
+            <div>
+              <span style={styles.lastUpdated}>{msg.lastUpdatedNoTime()}</span>
+              <span style={styles.lastUpdatedTime}> 8 hours ago</span>
```

Comment:
Would it make sense to put a "TODO" here instead since this may end up being a v2 feature?
https://codedotorg.atlassian.net/secure/RapidBoard.jspa?rapidView=26&projectKey=STAR&view=planning&selectedIssue=STAR-613